src/storybook/patterns/dropdown-in-modals/dropdown-in-modals.stories.mdx

```diff
@@ -0,0 +1,247 @@
+import { Canvas, Meta, Story } from "@storybook/addon-docs";
+import { useCallback, useState, useMemo, useRef } from "react";
+import { ComponentNameDecorator } from "../../components";
+import { OpenModalButton } from "./helpers";
+import ModalContent from "../../../components/Modal/ModalContent/ModalContent";
+import Dropdown from "../../../components/Dropdown/Dropdown";
+import Modal from "../../../components/Modal/Modal";
+import Flex from "../../../components/Flex/Flex";
+import Dialog from "../../../components/Dialog/Dialog";
+import Button from "../../../components/Button/Button";
+import { Link } from "vibe-storybook-components";
+import DialogContentContainer from "../../../components/DialogContentContainer/DialogContentContainer";
+import styles from "./dropdown-in-modals.module.scss";
+
+<Meta title="Technical Patterns/Dropdowns inside pop-overs" />
+
+<!--- Component documentation -->
+
+<ComponentNameDecorator withFoundationBackground>Display dropdowns inside pop-overs</ComponentNameDecorator>
+
+- [Overview](#overview)
+- [Up next](#up-next)
+- [Feedback](#feedback)
+
+## Problem description
+
+Our dropdowns are implemented using the React Select library.
+One of the styling issues with React Select dropdowns is that the menu is not fully displayed when its container has `overflow: hidden`, `overflow: auto`, or `overflow: scroll`.
+As a result, when dropdown menus are placed inside a scrolling container or within a Vibe's dialog or modal, the dropdown menu appears damaged, as you can see in the following examples.
+
+<Canvas>
+  <Story name="Modal with damaged dropdown">
+    {() => {
+      const [show, setShow] = useState(false);
+      const closeModal = useCallback(() => {
+        setShow(false);
+      }, [setShow]);
+      const options = [
+        { value: 1, label: "Option 1" },
+        { value: 2, label: "Option 2" },
+        { value: 3, label: "Option 3" }
+      ];
+      return (
+        <div>
+          <OpenModalButton setShow={setShow} />
+          <Modal title="Modal with dropdown" show={show} onClose={closeModal}>
+            <ModalContent>
+              <Dropdown options={options} />
+            </ModalContent>
+          </Modal>
+        </div>
+      );
+    }}
+  </Story>
+</Canvas>
+
+To address this problem, we need to adjust the styling of the dropdown menu to ensure it appears correctly even within these specific container.
+We implemented few methods for solving this issue.
+
+## 'insideOverflowContainer' prop
+
+To handle straightforward scenarios of displaying dropdowns inside a container with relevant overflow styles, you can utilize our dropdown's 'insideOverflowContainer' prop.
```

Comment:
Instead of 'insideOverflowContainer' try <code>insideOverflowContainer</code>, I think it would look better